frontend/public/components/secrets/create-secret/CreateConfigSubform.tsx

```diff
@@ -130,7 +130,12 @@ class CreateConfigSubformWithTranslation extends React.Component<
               </Button>
             </div>
           )}
-          <ConfigEntryForm id={index} entry={entryData.entry} onChange={this.onDataChanged} />
+          <PullSecretCredentialEntry
+            id={index}
+            entry={entryData.entry}
+            onChange={this.onDataChanged}
+            uid={entryData.uid}
+          />
```

Comment:
Also, from what I can tell, each entry's `auth` prop is simply calculated from username and password and never used or rendered in this sub-component. This means that it can be calculated in this component's `onDataChanged` function and omitted from `entryData.entry`.